lls/main.c

```diff
@@ -224,13 +254,50 @@ lls_scan(__attribute__((unused)) struct rte_timer *timer, void *arg)
 		lls_cache_scan(lls_conf, &lls_conf->nd_cache);
 }
 
+/* Send gratuitous ARP or ND announcement to fulfill LACP TX requirement. */
 static void
+lls_lacp_announce(__attribute__((unused)) struct rte_timer *timer, void *arg)
```

Comment:
@cjdoucette, could you read DPDK's bonding driver to see if we can call the transmitting function with no packets? I'm concerned that this uncommon behavior, namely, sending gratuitous packet often, can lead an IX to complain about Gatekeeper.